prow/cmd/checkconfig/main.go

```diff
@@ -829,6 +837,28 @@ func validateNeedsOkToTestLabel(cfg *config.Config) error {
 	return utilerrors.NewAggregate(queryErrors)
 }
 
+func validateManagedWebhooks(cfg *config.Config) error {
+	var errs []error
+	orgs := sets.String{}
+	for repo := range cfg.ManagedWebhooks {
```

Comment:
Does this make more sense as a CRD? Not sure why we need/want this to live in config.yaml

Reply:
I think it will make much more sense to make it a CRD, if we are going to develop a controller to reconcile the secrets. But as we synced with Aaron last time, he was mainly considering this approach for service-account keys and ssh private keys, which is a different use case and didn't consider it as a blocker to the current HMAC rotation design.